drivers/serial/uart_nrf_sw_lpuart.c

```diff
@@ -52,6 +56,22 @@ enum rx_state {
 	RX_TO_OFF,
 };
 
+struct lpuart_int_driven {
+	uart_irq_callback_user_data_t callback;
+	void *user_data;
+
+	uint8_t txbuf[CONFIG_NRF_SW_LPUART_INT_DRV_TX_BUF_SIZE];
+	uint32_t txlen;
+
+	uint8_t rxbuf[CONFIG_NRF_SW_LPUART_MAX_PACKET_SIZE];
+	uint32_t rxlen;
```

Comment:
`uint32_t rxlen;` is inconsistent with line 101, `size_t rx_len;`, same for tx

Reply:
fixed